featurebyte/service/catalog.py

```diff
@@ -22,51 +15,12 @@ class CatalogService(BaseDocumentService[CatalogModel, CatalogCreate, CatalogSer
 
     document_class = CatalogModel
 
-    async def _ensure_default_catalog_available(self) -> None:
```

Comment:
are these functions not required anymore?

Reply:
Yes, these functions overrides the one in the base class to automatically create a default catalog if it does not exist.
Since we no longer create any default catalog these are no longer needed.